scripts/debian-templ/rules

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/make -f
+
+# See debhelper(7) (uncomment to enable).
+# Output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
+
+# dpkg-buildflags sets link time optimisation by default. Because we want to
+# finish before the heat death of the universe, we disable the optimisation for
+# this package.
+export DEB_BUILD_MAINT_OPTIONS=optimize=-lto
+
+# See FEATURE AREAS in dpkg-buildflags(1).
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+# See ENVIRONMENT in dpkg-buildflags(1).
+# Package maintainers to append CFLAGS.
+#export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+# Package maintainers to append LDFLAGS.
+#export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
```

Comment:
Have you checked that the net effect of using Debian's default option set is reasonable compared with what our own makefiles are doing and that our own Makefile's options are taking effect as intended? Our options have obviously been carefully chosen although if it's possible to mix in Debian's hardening and optimisations without loss of performance and functionality that is obviously ideal ultimately, if not now.

Reply:
I haven't checked anything, I suppose I could check what our zf performance tests do and test that manually. Do you know if there is a way to unset all debian build flags? Neither google nor `man dpkg-buildflags` gave a good option beyond disabling all areas individually

Reply:
I don't know but initially shall we just compare the command lines from tgz build and deb build and see how flags differ?

Reply:
When I was tracking down the LTO stuff, which resulted in the disabling on line 10, I compared the output from `make` for the problematic `cc` command, but there was no difference. I guess it was being picked up from the environment variables.